x/distribution/client/cli/tx.go

```diff
@@ -195,25 +197,14 @@ $ %s tx distribution set-withdraw-addr cosmos1gghjut3ccd8ay0zduzj64hwre2fxs9ld75
 
 func NewFundCommunityPoolCmd(clientCtx client.Context) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "community-pool-spend [proposal-file]",
+		Use:   "fund-community-pool [amount]",
```

Comment:
is there a test case for this CLI cmd?

Reply:
There's an existing WIP PR for adding more cli_tests(https://github.com/cosmos/cosmos-sdk/pull/6349). Will ensure to add a test for this, once this PR is merged